db/migrations/00026_create_header_sync_transactions_table.sql

```diff
@@ -2,7 +2,7 @@
 CREATE TABLE header_sync_transactions (
   id          SERIAL PRIMARY KEY,
   header_id   INTEGER NOT NULL REFERENCES headers(id) ON DELETE CASCADE,
-  hash        VARCHAR(66),
+  hash        VARCHAR(66) UNIQUE,
```

Comment:
Spoke IRL about this but this will run into challenges when we have multiple `header_id`s for the same header, which can happen since we currently only treat a header row as unique if it shares the same block number, block hash, _and_ eth_node_fingerprint. We should probably just treat headers as unique by number and suggest that folks only point a vDB instance at one network at a time (unique by number + hash would enable syncing against mainnet + kovan, for example - but I think truly supporting that would entail deeper thinking)

Reply:
Yeah I'd not be willing to sync dual networks at all, no idea what'd happen :x